servicecontrol/creating-config-file.md

```diff
@@ -77,13 +77,17 @@ You can configure ServiceControl to forward any consumed messages into alternate
 
   The period that defines whether an endpoint is considered alive or not. Default: `00:00:40` (40 secs)
 
+* `ServiceControl/MaximumMessageThroughputPerSecond`

```

Comment:
well i think this config setting needs more context. since as it stands i would change it to 9999999 so it goes zooom